lua/lsp-inlayhints/handler_helper.lua

```diff
@@ -115,16 +115,20 @@ local render_hints = function(bufnr, parsed, namespace)
       virt_text = param_vt
     end
 
+    local padding = ""
     if config.options.inlay_hints.max_len_align then
-      virt_text = string.rep(
+      padding = string.rep(
         " ",
         max_len - current_line(bufnr, line):len() + config.options.inlay_hints.max_len_align_padding
-      ) .. virt_text
+      )
     end
 
     if virt_text ~= "" then
       vim.api.nvim_buf_set_extmark(bufnr, namespace, line, 0, {
-        virt_text = { { virt_text, config.options.inlay_hints.highlight } },
+        virt_text = {
+          config.options.inlay_hints and { padding, "NONE" },
```

Comment:
Doesn't the left side always evaluate to `true`?

Reply:
Oops, thats a typo, should be checking if max_len_align is set